app/src/main/java/ch/epfl/skysync/database/tables/UserTable.kt

```diff
@@ -57,6 +61,59 @@ class UserTable(db: FirestoreDatabase) : Table<User, UserSchema>(db, UserSchema:
     }
   }
 
+  /**
+   * Returns the available user on the given day and timeslot
+   *
+   * @param localDate The requested day
+   * @param timeslot The requested timeslot
+   * @param onError Callback called when an error occurs
+   */
+  suspend fun getUsersAvailableOn(
```

Comment:
I'd personally prefer to have this function at the end of the class (so that we now we have query methods at the end by convention)

Reply:
I will change it